en/editor/publish/custom-project-build-template.md

```diff
@@ -1,67 +1,72 @@
+| **WeChat Mini Game** | wechatgame | `game.ejs`, `game.json`, `project.config.json` |
+| **Web Desktop** | web-desktop | `index.ejs` |
+| **Web Mobile** | web-mobile | `index.ejs` |
+| **Native** | native | not supported yet |
+
+## Customizing `application.js`
+
+All platforms generate a startup script `application.js` after the build. To customize the startup script, there are two ways:
 
-## Custom Build Plugins
+- Refer to the way described at the beginning of this article, place `application.js` in the specified directory and then customize it as needed.
+- Click **Project -> Create Build Template** in Creator's top menu bar, then select **application.ejs** to customize the generated `application.ejs` file, and the generated directory will be displayed in the **Console** panel. The `application.ejs` file will be compiled into an `application.js` file when built.
 
-It is currently in the internal testing phase and is not open to the public at this time.
+The directory where the `application.ejs` file is located determines the platform on which it will take effect.
 
-## Customize application.js
+- If it is placed in the `build-templates/common` directory, it will take effect for all platforms.
+- If it is placed in the `build-templates/{platform}` directory, it will take effect for the specified platform. The `application.ejs` file in this directory is used first when building.
 
-`application.js` is the startup script generated by the build that is the same for all platforms. If you need to customize it, you can place `application.js` in the specified directory in the manner described above to customize. Or click **Project -> Create project build template -> Create application.ejs** in the main menu to generate the corresponding ejs file, and modify it in it. The `application.ejs` placed in the `build-templates/common` folder will work for all build platforms. Placed in the `build-templates/{platform}` directory will only work for the specified platform. By default, the files in the platform's build path are used first for the build. Using ejs, you can avoid that after checking the MD5 Cache option, some file path changes cause the customized `application.js` to become unavailable. However, it should be noted that because the startup script and the engine interface are strongly related, it cannot be completely guaranteed to remain the same in the upgrade iteration of the major version. If there is a modification, we will mention it in the update log. Please pay attention after the upgrade version.
+Using the `ejs` method of customization prevents the custom `application.js` from becoming unavailable if the MD5 Cache option is checked during the build. However, it should be noted that since the startup scripts and the engine interface are strongly correlated, it is not possible to ensure that they remain completely unchanged during major iterations of the upgrade, so we will mark any changes in the changelog.
```

Comment:
```suggestion
Using the `ejs` method of customization prevents the custom `application.js` from becoming unavailable if the MD5 Cache option is checked during the build. However, it should be noted that since the startup scripts and the engine interface are strongly correlated, it is not possible to ensure that they remain completely unchanged during major iterations of the upgrade, so we will mark any changes in the changelog.
-->
```